Mail/Views/Thread/Message/MessageBodyView.swift

```diff
@@ -108,23 +105,26 @@ struct MessageBodyView: View {
         }
     }
 
-    private func loadBody(blockRemoteContent: Bool) {
+    private func loadBody(presentableBody: PresentableBody, blockRemoteContent: Bool) {
+        let uid = messageUid
```

Comment:
Is there a reason to create the `uid` variable here?
If so, can we use a more explicit name to understand the reason why?